src/PrestaShopBundle/Resources/config/services/core/kpi.yml

```diff
@@ -52,3 +52,11 @@ services:
         - '@prestashop.adapter.kpi.net_profit_per_visit'
       - '@prestashop.core.hook.dispatcher'
       - 'orders'
+
+  PrestaShop\PrestaShop\Core\Kpi\Row\HookableKpiRowFactory:
+    arguments:
+      - - '@prestashop.adapter.kpi.pending_discussion_threads'
```

Comment:
Looks like there's a typo here

Reply:
no he is probably talking about double `- -` the beginning of arguments

Reply:
its array so thats how it should be. 
      - - '@prestashop.adapter.kpi.conversion_rate'
        - '@prestashop.adapter.kpi.abandoned_cart'
        - '@prestashop.adapter.kpi.average_order_value'
        - '@prestashop.adapter.kpi.net_profit_per_visit'

Reply:
oh, right, okay :+1: